src/src/services/hasura/hasura.service.ts

```diff
@@ -381,4 +381,51 @@ export class HasuraService {
 			return { [tableName]: result };
 		}
 	};
+
+	public async getDataFromSQL(sql: string) {
```

Comment:
can we move this to hasura service?
rename as
`public async executeRawSql(sql: string) {`

Reply:
Resolved in Yogesh's PR